types/wx-js-sdk-dt/index.d.ts

```diff
@@ -0,0 +1,468 @@
+// Type definitions for wx-js-sdk 1.2
+// Project: https://res.wx.qq.com/open/js/jweixin-1.2.0.js
+// Definitions by: Bian Zhongjie <https://github.com/agasbzj>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+export namespace wx {
```

Comment:
I added /// <reference types="wx-js-sdk-dt"> in my test file and it says:
ERROR: 2:1  expect  TypeScript compile error: Invalid 'reference' directive syntax.
ERROR: 4:1  expect  TypeScript compile error: Cannot find name 'wx'.
The target js lib can only be downloaded at run time, there is no npm packaged for it. How to start my test?